test/common/http/header_utility_test.cc

```diff
@@ -671,6 +751,30 @@ invert_match: true
   EXPECT_FALSE(HeaderUtility::matchHeaders(matching_headers, header_data));
 }
 
+TEST(MatchHeadersTest, HeaderContainsMatchIgnoreCase) {
+  for (const auto& match_value : {"onevalue", "OneValue"}) {
```

Comment:
Why is this a `for` loop? Here and other cases.

Reply:
not needed now.